kotlin/create-symbol-styles-from-web-styles/src/main/java/com/esri/arcgisruntime/sample/createsymbolstylesfromwebstyles/MainActivity.kt

```diff
@@ -0,0 +1,239 @@
+package com.esri.arcgisruntime.sample.createsymbolstylesfromwebstyles
+
+import android.graphics.Color
+import android.os.Bundle
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.BasemapStyle
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.symbology.SymbolStyle
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.legend_row.view.*
+
+class MainActivity : AppCompatActivity() {
+
+    private val TAG = this::class.java.simpleName
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // authentication with an API key or named user is required to access basemaps and other
+        // location services
+        ArcGISRuntimeEnvironment.setApiKey(BuildConfig.API_KEY)
+
+        // create a unique value renderer
+        val uniqueValueRenderer = UniqueValueRenderer().apply {
```

Comment:
i think it's helpful here to have a comment about how this field name links to the data: the following is pinched from the Java implementation

```
      // add the name of a field from the feature layer data that symbols will be mapped to
```